pkg/commands/process/settings/rules/javascript/lang/jwt.yml

```diff
@@ -0,0 +1,45 @@
+type: risk
+patterns:
+  - pattern: |
+      jwt.sign($<DATA_TYPE>)
+    filters:
+      - variable: DATA_TYPE
+        detection: datatype
+languages:
+  - javascript
+trigger: local
+severity:
+  default: low
+  PII: critical
+  PHI: medium
+  PD: high
+skip_data_types:
+  - "Unique Identifier"
+metadata:
+  description: "Do not store sensitive data in jwt."
+  remediation_message: |
+    ## Description
+
+    JWTs are not a secure place to store sensitive data. This rule looks for any sensitive data types saved to a JWT.
+
+    ## Remediations
+
+    ❌ Avoid storing sensitive data in JWT:
+
+    ```javascript
+      const jwt = require('jsonwebtoken');
+      const token = jwt.sign({ user: { email: 'jhon@gmail.com' }});
+    ```
+
+    ✅ If you need to store user's information, use their unique database identifier instead of personal identifiable information:
+
+    ```javascript
+      const jwt = require('jsonwebtoken');
+      const token = jwt.sign({ user: user.uuid });
+    ```
+
+    <!--
+    ## Resources
+    Coming soon.
```

Comment:
@markmichon happy to have you review this new one :)

Reply:
resolving this for now so I can merge it, lets open a separate pr for any javascript docs related changes.